src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -3139,124 +3147,123 @@ public synchronized InputStream getInputStream(Range range, RendererConfiguratio
 				try {
-					Thread.sleep(1000);
+					Thread.sleep(params.waitbeforestart);
 				} catch (InterruptedException e) {
 					LOGGER.error(null, e);
 				}
 
-				if (newExternalProcess == null) {
-					LOGGER.trace("External process instance is null... sounds not good");
+				LOGGER.trace("Finished sleeping for " + params.waitbeforestart + " milliseconds");
+			}
+		} else if (
+			params.timeseek > 0 &&
+			media != null &&
+			media.isMediaparsed() &&
+			media.getDurationInSeconds() > 0
+		) {
+			// Time seek request => stop running transcode process and start a new one
+			LOGGER.debug("Requesting time seek: " + params.timeseek + " seconds");
+			params.minBufferSize = 1;
+			Runnable r = new Runnable() {
+				@Override
+				public void run() {
+					externalProcess.stopProcess();
 				}
+			};
 
-				externalProcess = newExternalProcess;
+			new Thread(r, "External Process Stopper").start();
+			lastStartSystemTime = System.currentTimeMillis();
+			ProcessWrapper newExternalProcess = player.launchTranscode(this, media, params);
+			try {
+				Thread.sleep(1000);
+			} catch (InterruptedException e) {
+				LOGGER.error(null, e);
 			}
 
-			if (externalProcess == null) {
-				return null;
+			if (newExternalProcess == null) {
+				LOGGER.trace("External process instance is null... sounds not good");
 			}
 
-			InputStream is = null;
-			int timer = 0;
-			while (is == null && timer < 10) {
-				is = externalProcess.getInputStream(low);
-				timer++;
-				if (is == null) {
-					LOGGER.debug("External input stream instance is null... sounds not good, waiting 500ms");
-					try {
-						Thread.sleep(500);
-					} catch (InterruptedException e) {
-					}
-				}
-			}
+			externalProcess = newExternalProcess;
+		}
 
-			// fail fast: don't leave a process running indefinitely if it's
-			// not producing output after params.waitbeforestart milliseconds + 5 seconds
-			// this cleans up lingering MEncoder web video transcode processes that hang
-			// instead of exiting
-			if (is == null && !externalProcess.isDestroyed()) {
-				Runnable r = new Runnable() {
-					@Override
-					public void run() {
-						LOGGER.error("External input stream instance is null... stopping process");
-						externalProcess.stopProcess();
-					}
-				};
+		if (externalProcess == null) {
+			return null;
+		}
 
-				new Thread(r, "Hanging External Process Stopper").start();
+		InputStream is = null;
+		int timer = 0;
+		while (is == null && timer < 10) {
+			is = externalProcess.getInputStream(low);
+			timer++;
+			if (is == null) {
+				LOGGER.debug("External input stream instance is null... sounds not good, waiting 500ms");
+				try {
+					Thread.sleep(500);
+				} catch (InterruptedException e) {
+				}
 			}
+		}
+
+		// fail fast: don't leave a process running indefinitely if it's
+		// not producing output after params.waitbeforestart milliseconds + 5 seconds
+		// this cleans up lingering MEncoder web video transcode processes that hang
```

Comment:
**T**his

Reply:
@Sami32 You keep commenting on things I didn't write. I have shuffled some code around to organize things, that doesn't mean I cleaned up every comment in that code 😉

Reply:
Sorry, from Github i saw that it was green, meaning changed, though it could have been only formatting. Just ignore it then ;)

Reply:
@Sami32 GitHub (or Git) often misinterprets what's changed in more complex diffs. I've found that most of the confusion comes from whitespace comparison, which is one of the reasons why the diffs in TortoiseGit are so much better. There you can easily switch whitespace comparison on and off.

That said, I sometimes move code which will make it a change without it being a "bug". In this case though, it's just an indentation change. If you look at the diff for DLNAResource for 6f42f3a75d3041178aa02bdee6dd41ee91dde98c in TortoiseGit and turn off whitespace comparison, you will see it easily. All I did was to remove a set of parenthesis.